deployments/helm-chart/templates/controller-appprotect-dos-arb.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.controller.appprotectdos.enable }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "nginx-ingress.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "nginx-ingress.appProtectDosArbName" . }}
+    spec:
+      containers:
+        - name: {{ include "nginx-ingress.appProtectDosArbName" . }}
+          image: "{{ .Values.controller.appprotectdos.arbitrator.repository }}"
+          imagePullPolicy: IfNotPresent
+          resources:
+            limits:
+              memory: "128Mi"
+              cpu: "500m"
+          ports:
+            - containerPort: 3000
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: svc-appprotect-dos-arb
```

Comment:
because this name is static, it creates a problem with multiple Ingress Controller deployments (helm releases) are installed in the same namespace, which is supported installation type. https://docs.nginx.com/nginx-ingress-controller/installation/running-multiple-ingress-controllers/  this way, we can have multiple Ingress Controllers where each Ingress Controller handles resources only of its IngressClass.  The static name will prevent helm from installing more than one helm release in one namespace :( 

It also looks like the DOS module doesn't support configuring the DNS name of the Arbitrator service :( 

can we support the described installation type?

Reply:
Adding new helm chart for arbitrator